apps/typegpu-docs/src/content/examples/algorithms/mnist-inference/index.html

```diff
@@ -0,0 +1,128 @@
+<canvas></canvas>
+<div class="predictions-container">
+  <div class="predictions-label">Predictions:</div>
+
+  <div class="bars-container">
+    <div class="bar-container">
+      <div class="bar-label">0</div>
+      <div class="bar">
+        <div class="bar-fill" id="bar-fill-0" style="width: 0%"></div>
+      </div>
+    </div>
```

Comment:
I am challenging you to reduce these 3 level deep trees into just one `div` each 🥇🏃‍♂️. I am fairly sure you can use `querySelectorAll` instead of numeric ids, `::before` or `::after` for the bar fill, and a CSS variable for the width of the fill. Good luck 🫡.

```suggestion
    <div class="bar"></div>
```